census/ui_app/HelperFields.jsx

```diff
@@ -66,6 +66,8 @@ const SubmitActions = props => {
       <div className="comments"></div>
     </div>
     </div>);
+  } else if (props.isReview && !props.canReview) {
```

Comment:
Ah, yes, I see what you mean. I think I'd like to leave it in to clarify my intension though.